packages/app/src/app/pages/Pro/index.tsx

```diff
@@ -237,35 +237,45 @@ const Expiring = ({
   user,
   updateSubscriptionClicked,
   isUpdatingSubscription,
-}) => (
-  <MaxWidth width={500}>
-    <Centered horizontal>
-      <Avatar src={user.avatarUrl} />
-      <Badge type="pro" style={{ opacity: 0.8 }}>
-        Pro
-      </Badge>
-      <Heading>Your subscription is expiring</Heading>
+}) => {
+  // if user has yearly subscription show the date he created the subscription and since it's passed they know it's next year
+  // if Monthly show the day he created and the next month on the calendar
+  const since = new Date(user.subscription.since);
+  const expiringDate =
+    user.subscription.duration === 'yearly'
+      ? format(since, 'do MMM')
+      : `${format(since, 'do')} ${format(addMonths(new Date(), 1), 'MMM')}`;
```

Comment:
Would this also work when we're in February? So eg. when your `since` is `Jan 12`, and it's `Feb 3` that the user sees this message, will it say `Feb 12`?

Reply:
It wopuldn't yeah

Need to check if the day they subscribed is larger than the day we are and only then show the next month